vcr/stubs/httpx_stubs.py

```diff
@@ -108,7 +108,8 @@ def _play_responses(cassette, request, vcr_request, client, kwargs):
         if not next_url:
             break
 
-        vcr_request = VcrRequest("GET", next_url, None, dict(response.headers))
+        method = request.method if response.status_code != 303 else "GET"
```

Comment:
Its tricky to know what would be the best logic here because it is not strictly mandated by the HTTP spec.

However according to https://developer.mozilla.org/en-US/docs/Web/HTTP/Redirections the only status code that *always* changes to `GET` method is `303`. So this LGTM!